packages/cice5/package.py

```diff
@@ -25,45 +27,64 @@ class Cice5(MakefilePackage):
     depends_on("mpi")
     depends_on("netcdf-fortran@4.5.2:")
     depends_on("netcdf-c@4.7.4:")
-    # TODO: For initial verification we are going to use static pio.
-    #       Eventually we plan to move to shared pio
-    # ~shared requires: https://github.com/spack/spack/pull/34837
-    depends_on("parallelio~pnetcdf~timing~shared")
+    
     depends_on("datetime-fortran")
     depends_on("oasis3-mct+deterministic", when="+deterministic")
     depends_on("oasis3-mct~deterministic", when="~deterministic")
-    depends_on("libaccessom2+deterministic", when="+deterministic")
-    depends_on("libaccessom2~deterministic", when="~deterministic")
+
+
+    with when("@master"):
```

Comment:
Use what's done in MOM5: `with when("@:access-esm0,access-esm2:"):`

Reply:
resolved in https://github.com/ACCESS-NRI/spack-packages/pull/179